static_defaults.go

```diff
@@ -0,0 +1,10 @@
+package rdns
+
+var (
+	DoQPort      string = ":8853"
```

Comment:
No need for the colon if your function uses https://pkg.go.dev/net#JoinHostPort from the standard library